edit/provisioning.cattle.io.cluster/MachinePool.vue

```diff
@@ -156,7 +159,33 @@ export default {
       <portal-target :name="'advanced-' + uuid" multiple />
 
       <div class="spacer" />
-
+      <div class="row">
+        <div class="col span-4">
+          <h3>
+            {{ t('cluster.machinePool.autoReplace.label') }}
+          </h3>
+          <UnitInput
+            v-model.number="unhealthyNodeTimeoutInteger"
+            :placeholder="t('containerResourceLimit.cpuPlaceholder')"
+            :mode="mode"
+            :output-modifier="true"
+            :base-unit="t('cluster.machinePool.autoReplace.unit')"
+            :tooltip="t('cluster.machinePool.autoReplace.toolTip')"
```

Comment:
Now that the label text is no longer inline with the input, I'm wondering if it will make more sense to include the tooltip next to the label, but it seems that we might prefer tooltips inside text inputs after finding an example.. 

![image](https://user-images.githubusercontent.com/835961/153240098-ce3c4b28-0101-476e-add4-85a97e89cb92.png)

I don't think there's anything actionable that needs to be done here, but I'm interested in learning what the patterns are for placement of tooltips.

Reply:
Lauren confirmed that it would look better in this case if all the tooltips were consistently next to the headers. Basically, if there is a header, then the tooltip should go with the header.